src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -272,12 +288,40 @@ def generate_IPR_2_default_config(self, filename):
         for key in device_b
         if key not in reg_conditional_keys
     }
+    conditionals_d = {key: device_d[key] for key in conditional_keys}
+    device_d = {
+        key: device_d[key]
+        for key in device_d
+        if key not in reg_conditional_keys
+    }
+    conditionals_h = {key: device_h[key] for key in conditional_keys}
+    device_h = {
+        key: device_h[key]
+        for key in device_h
+        if key not in reg_conditional_keys
+    }
 
     # Load grant requests.
     grant_a = json.load(
         open(os.path.join('testcases', 'testdata', 'grant_0.json')))
     grant_b = json.load(
         open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c['operationParam']['operationFrequencyRange']['lowFrequency'] = 3610000000
```

Comment:
Don't you want these using 3620-3630, since they are near East21 which is activated on that channel?

Reply:
ok.